scripts/parse-retrievable-deals.js

```diff
@@ -102,12 +103,15 @@ function * processDeal (deal) {
   assert.strictEqual(typeof StartEpoch, 'number', `StartEpoch is not a number: ${JSON.stringify(deal.Proposal)}`)
   const started = StartEpoch * BLOCK_TIME + GENESIS_TS
 
+  // Skip deals that are less than 24 hours old, to account for IPNI ingestion delays
+  if (Date.now() - started < ONE_DAY_IN_MILLISECONDS) return
```

Comment:
we will retry ingesting these deals later, right?

Reply:
Yes. As @pyropy wrote in his comment:

> Based on lotus docs and your blog post I assume `StateMarketDeals` snapshot stores all deals starting from genesis, so in case we do skip a deal because of it's age it will still be present in the snapshot we download next time.

We run fil-deal-ingester daily. Deals skipped today will be ingested tomorrow.